### PR TITLE
[UI] - Adding logic to enable init.sh execution on Alpine Linux

### PR DESCRIPTION
**Pull request by izaac**
### Summary
First part of https://github.com/rancher/qa-tasks/issues/1549

### Occurred changes and/or fixed issues
Adding logic to execute the Jenkins CI initialization script on Alpine Linux images. This allows the execution on the new runners.
Explicitly set the `GOROOT` to the downloaded golang extracted tar.
Everything else is shell agnostic and runs transparently in then `ash` shell.

### Technical notes summary
Alpine Linux isn't binary compatible by default with the golang Linux binaries as it ships with the C lib `musl`. The go binaries require `glibc`. Adding `gcompat` fix this.

### Areas or cases that should be tested
Jenkins

### Checklist
- [x] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [x] The PR has a Milestone 
- [x] The PR template has been filled out
- [x] The PR has been self reviewed 
- [x] The PR has a reviewer assigned
- [x] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [ ] ~~The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes~~

### CODE REVIEW

File: cypress/jenkins/init.sh [hunk 1]
```diff
@@ -3,6 +3,10 @@
 set -x
 set -e
 
+if cat /etc/os-release | grep -iq "Alpine Linux"; then
+ apk update && apk add --no-cache gcompat g++ make alpine-sdk
```
Comment (rancher-max) [inline cypress/jenkins/init.sh:7]: I assume we are, but would be remiss not to ask... are we using `g++`, `make`, and `alpine-sdk` somewhere as well that are required to install here?

**Review by rancher-max — APPROVED** [review]

File: cypress/jenkins/init.sh (hunk 1 above)
Comment (izaac) [reply cypress/jenkins/init.sh:7]: Yes, we need make. We use it down the line in the script.

**Review by yonasberhe23 — APPROVED** [review]

File: cypress/jenkins/init.sh (hunk 1 above)
Comment (izaac) [reply cypress/jenkins/init.sh]: I am going to test if we actually need the SDK, but the others are a must.

Comment (izaac) [reply cypress/jenkins/init.sh]: @rancher-max good eye 👍🏽 the sdk isn't needed.

**Review by rancher-max — APPROVED** [review]

**Pull request merged**